src/main/java/com/iridium/iridiumskyblock/BackButton.java

```diff
@@ -0,0 +1,14 @@
+package com.iridium.iridiumskyblock;
+
+import com.iridium.iridiumcore.Item;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class BackButton {
+
+    public boolean enabled;
+    public String command;
```

Comment:
I think this should be an Inventory instead of a command, then we can optionally paste the Back button if that inventory is null or not